ts/DataGrid/DataGrid.ts

```diff
@@ -517,13 +538,15 @@ class DataGrid {
 
 
             for (let k = 0; k < columnsInPresentationOrder.length; k++) {
```

Comment:
Would be better to replace it with `let k = 0; kEnd = columnsInPresentationOrder.length; k < kEnd` - performance reasons.